integration_tests/mockApis/cancellation.ts

```diff
@@ -14,25 +16,52 @@ export default {
     stubFor({
       request: {
         method: 'POST',
-        url: `/premises/${args.premisesId}/bookings/${args.bookingId}/cancellations`,
+        url: paths.cas3.premises.bookings.cancellations.create({
+          premisesId: args.premisesId,
+          bookingId: args.bookingId,
+        }),
       },
       response: {
         status: 201,
         headers: { 'Content-Type': 'application/json;charset=UTF-8' },
         jsonBody: args.cancellation,
       },
     }),
-  stubCancellationGet: (args: { premisesId: string; bookingId: string; cancellation: Cancellation }) =>
+  stubCancellationCreateErrors: (args: { premisesId: string; bookingId: string; params: Array<string> }) =>
+    stubFor(
+      errorStub(
+        args.params,
+        paths.cas3.premises.bookings.cancellations.create({ premisesId: args.premisesId, bookingId: args.bookingId }),
+        'POST',
+      ),
+    ),
+  verifyCancellationCreate: async (args: { premisesId: string; bookingId: string; cancellation: Cancellation }) =>
+    (
+      await getMatchingRequests({
+        method: 'POST',
+        url: paths.cas3.premises.bookings.cancellations.create({
+          premisesId: args.premisesId,
+          bookingId: args.bookingId,
+        }),
+      })
+    ).body.requests,
+  stubCancellationReferenceData: () => stubFor(cancellationReasons),
+}
+
+const cas3v2ApiDisabledStubs = {
+  stubCancellationCreate: (args: {
+    premisesId: string
+    bookingId: string
+    cancellation: Cancellation
+  }): SuperAgentRequest =>
     stubFor({
       request: {
-        method: 'GET',
-        url: `/premises/${args.premisesId}/bookings/${args.bookingId}/cancellations/${args.cancellation.id}`,
+        method: 'POST',
```

Comment:
The diff view is a bit out of whack here -- this should map to the old `stubCancellationCreate` above!